controllers/chat.js

```diff
@@ -9,6 +9,13 @@ const createChat = async (req, res) => {
   res.status(201).json(chat)
 }
 
+const getChats = async (req, res) => {
+  const chats = await chatService.getChats(req.user)
```

Comment:
add filtering chats by user name

Reply:
and I would extract all the necessary data from the req.user and then transfer them to the service, as we do this everywhere on the project